articles/storage/files/storage-files-identity-auth-domain-services-enable.md

```diff
@@ -194,9 +194,9 @@ There are five Azure built-in roles for Azure Files, some of which allow grantin
 
 - **Storage File Data Share Reader** allows read access in Azure file shares over SMB.
 - **Storage File Data Privileged Reader** allows read access in Azure file shares over SMB by overriding existing Windows ACLs.
-- **Storage File Data Share Contributor** allows read, write, and delete access in Azure file shares over SMB.
-- **Storage File Data Share Elevated Contributor** allows read, write, delete, and modify Windows ACLs in Azure file shares over SMB.
-- **Storage File Data Privileged Contributor** allows read, write, delete, and modify Windows ACLs in Azure file shares over SMB by overriding existing Windows ACLs.
+- **Storage File Data SMB Share Contributor** allows read, write, and delete access in Azure file shares over SMB.
+- **Storage File Data SMB Share Elevated Contributor** allows read, write, delete, and modify Windows ACLs in Azure file shares over SMB.
+- **Storage File Data SMB Privileged Contributor** allows read, write, delete, and modify Windows ACLs in Azure file shares over SMB by overriding existing Windows ACLs.
```

Comment:
```suggestion
- **Storage File Data Privileged Contributor** allows read, write, delete, and modify Windows ACLs in Azure file shares over SMB by overriding existing Windows ACLs.
```

Reply:
@khdownie Ah, good point. I also missed the `SMB` part for the Reader role. To avoid confusion when comparing the roles with the portal I changed the order of the roles to how they appear in the portal.